plugins/net_plugin/net_plugin.cpp

```diff
@@ -2203,6 +2208,18 @@ namespace eosio {
               sync_next_expected_num < sync_last_requested_num );
    }
 
+   bool sync_manager::is_sync_request_ahead_allowed(block_num_type blk_num) const REQUIRES(sync_mtx) {
```

Comment:
Not a big deal. It would be clearer in the `if` conditions if `blk_num` is renamed to `next_requested_block_num`.